api-reference/10 UI Components/GridBase/1 Configuration/columnChooser/position.md

```diff
@@ -0,0 +1,157 @@
+---
+id: GridBase.Options.columnChooser.position
+type: PositionConfig
+default: undefined
+---
+---
+##### shortDescription
+Configures the column chooser's position.
+
+---
+You can specify the [my](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/my/), [at](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/at/), and [of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) properties to position a column chooser. 
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function(){
+        $("#{widgetName}").dx{WidgetName}({
+            // ...
+            columnChooser: {
+                // ...
+                position: {
+                    my: "left",
+                    at: "right",
+                    of: ".dx-datagrid-column-chooser-button"
+                }
+            },
+        })
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-{widget-name} ... >
+        <dxo-column-chooser ... >
+            <dxo-position
+                my="left"
+                at="right"
+                of=".dx-datagrid-column-chooser-button">
```

Comment:
do you need to use ".dx-datagrid-column-chooser-button" class here? Shouldn't it be something like ".dx-{widgetname}-column-chooser-button"